samples/nrf9160/gnss/Kconfig

```
@@ -182,6 +182,16 @@ config GNSS_SAMPLE_ASSISTANCE_REFERENCE_ALT
 
 endif # GNSS_SAMPLE_ASSISTANCE_MINIMAL && GNSS_SAMPLE_LOW_ACCURACY
 
+if NRF_CLOUD_PGPS && !NRF_CLOUD_PGPS_PARTITION
+
+config GNSS_SAMPLE_PGPS_MCUBOOT_PARTITION
+	bool "Use mcuboot secondary partition for P-GPS data"
```

This isn't supposed to be configured by the user, so maybe we could remove the prompt?
```suggestion
	bool
```